### ISSUE TITLE
Update default version?

### ISSUE THREAD

**Issue by moschlar** [toscawidgets/tw2.jquery#3]
I just added the js files for jQuery 1.8.3 and 1.9.1, should we consider updating the default version?

This would probably require checking all `tw2.jqplugin.*` for compatibility with the new versions, so it might be a lot of work...

**Comment by techtonik** [comment]
Yes, plz. Current version gives deprecation warning in Firefox, so at least 1.11.3 is needed.

https://stackoverflow.com/questions/26177305/use-of-getpreventdefault-is-deprecated-use-defaultprevented-instead-why-im/47951176#47951176
